src/vs/vscode.proposed.d.ts

```diff
@@ -168,14 +168,19 @@ declare module 'vscode' {
 		availableFormats: ColorFormat[];
 
 		/**
+		 * Controls whether the color decorator is rendered.
+		 */
+		renderDecorator: boolean;
```

Comment:
> Give the decision to extensions: 

Disagree, we control the UI and the extensions just provide the data. I think a core experience of our editor is that it always behaves the same, no matter in what language you are coding.